README.md

```diff
@@ -11,10 +11,14 @@
 
 ## Overview	
 
-Penney's game, named after its inventor Walter Penney, is a binary (head/tail) sequence generating game between two or more players. Player A selects a sequence of heads and tails (of length 3 or larger), and shows this sequence to player B. Player B then selects another sequence of heads and tails of the same length. Subsequently, a fair coin is tossed until either player A's or player B's sequence appears as a consecutive subsequence of the coin toss outcomes. The player whose sequence appears first wins [[Wikipedia](https://en.wikipedia.org/wiki/Penney%27s_game)].
-							
+Penney's game, named after its inventor Walter Penney, is a binary (head/tail) sequence generating game between two or more players. Player A selects a sequence of heads and tails (of length 3 or larger), and shows this sequence to player B. Player B then selects another sequence of heads and tails of the same length. Subsequently, a fair coin is tossed until either player A's or player B's sequence appears as a consecutive subsequence of the coin toss outcomes. The player whose sequence appears first wins.							
 Here we have a clone of this game that I wrote during coronavirus quarantine days.
 
+ℹ️ In playing with computer, the computer selects its sequence in a completely random way
+
+ℹ️ For details about how the winning probability is calculated, you can visit [here](https://arxiv.org/pdf/1212.3973)
```

Comment:
`[Penney’s game between many players](https://arxiv.org/pdf/1212.3973) paper`

+ better naming than `here`

Reply:
I have also included this text in the `print_description` function